phy/stats/ccg.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+"""Cross-correlograms."""
+
+#------------------------------------------------------------------------------
+# Imports
+#------------------------------------------------------------------------------
+
+import numpy as np
+
+from ..ext import six
+from ..utils.array import _index_of, _unique
+
+
+#------------------------------------------------------------------------------
+# Cross-correlograms
+#------------------------------------------------------------------------------
+
+def _increment(arr, indices):
+    """Increment some indices in a 1D vector of non-negative integers.
```

Comment:
hmmm fair enough. I wonder whether we could create ```d_assert```, ```d_ae``` functions or something (i.e. debug_assert) - if they called a function which checks whether ```DEBUG == True``` then would they get optimised away? Or would there still be an overhead due to context switch or something? In C the compiler would just optimise that away altogether if DEBUG is set to True...